doc/whats_new/v0.20.rst

```diff
@@ -591,9 +591,19 @@ Support for Python 3.3 has been officially dropped.
   :issue:`11082` by :user:`Sadhana Srinivasan <rotuna>`
   and :user:`Hanmin Qin <qinhanmin2014>`.
 
+
+- :class:`preprocessing.PolynomialFeatures` now supports sparse input.
```

Comment:
I think we'll include this in v0.21 and not 0.20 (since it's kind of a new feature), please move the entry to the corresponding file.

Reply:
This is a merge error. Use `git checkout master doc/whats_new/v0.20.rst; git commit`

Reply:
This still needs to be addressed.